tests/resources/xfra.csv

```diff
@@ -7541,3 +7544,23 @@
 2019-09-03 00:00:00+00:00,2019-09-03 07:01:00,2019-09-03 15:30:00
 2019-09-04 00:00:00+00:00,2019-09-04 07:01:00,2019-09-04 15:30:00
 2019-09-05 00:00:00+00:00,2019-09-05 07:01:00,2019-09-05 15:30:00
+2019-09-06 00:00:00+00:00,2019-09-06 07:01:00,2019-09-06 15:30:00
```

Comment:
what is causing these 2019 additions to each calendar's csv?

Reply:
I think the calendars go a year out, so when you regenerate this output it adds the days out to a year from now. I can remove these if we want.